src/app/images-collection/images-collection-detail/images-collection-detail.component.ts

```diff
@@ -148,6 +152,7 @@ export class ImagesCollectionDetailComponent implements OnInit, AfterViewInit {
     return this.getImagesCollection().pipe(
       map(imagesCollection => {
         this.imagesCollection = imagesCollection;
+        this.sourceCatlaogLink = urljoin(this.appConfigService.getConfig().catalogUiUrl, this.imagesCollection.sourceCatalog);
```

Comment:
There should be an `if (this.imagesCollection.sourceCatalog)` before constructing the link